gapic/packaging/dependencies.yaml

```diff
@@ -91,6 +91,9 @@ google-iam-v1_version:
   ruby:
     name_override: grpc-google-iam
     lower: '0.6.8'
+  java:
+    name_override: grpc-google-iam-v1
```

Comment:
I agree that currently it is feasible for Java to do it in the transformer. But it looks risky to me that we assume it is always correct for Java to append 'grpc-' to the default name. This introduces the extra dependency of the default name in gapic. I feel we should follow the existing overriding method here as other languages.